include/ck_tile/host/reference/reference_batched_transpose.hpp

```diff
@@ -56,4 +73,45 @@ CK_TILE_HOST void reference_batched_transpose(const HostTensor<Type>& x,
 
     make_ParallelTensorFunctor(f, N)(std::thread::hardware_concurrency());
 }
+// different threshold for different dtype
+template <typename DataType>
+inline auto batched_transpose_get_elimit(std::string /*init_method*/)
```

Comment:
This header is for the definition of `reference_*()` functions. You have to put application-specific configurations into their example folders.